<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1081</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 87, 83);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QPushButton" name="calibrationbutton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>231</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 238, 236);</string>
    </property>
    <property name="text">
     <string extracomment="Generate intrinsic camera parameter">カメラキャリオブレーション 
Camera Calibration</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generateimagebutton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>240</y>
      <width>231</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 238, 236);</string>
    </property>
    <property name="text">
     <string extracomment="Capture iamages for training data">学習データ記録
Generate Training Image</string>
    </property>
   </widget>
   <widget class="Line" name="line_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>340</y>
      <width>1901</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="P66832A030Pbutton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>231</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 238, 236);</string>
    </property>
    <property name="text">
     <string>66832A030P COWLTOP</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>350</y>
      <width>1901</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(238, 238, 236);</string>
    </property>
    <property name="text">
     <string>部品リスト</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(238, 238, 236);</string>
    </property>
    <property name="text">
     <string>キャリブレーシオン又は点検など</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="checkarucobutton">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>240</y>
      <width>231</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 238, 236);</string>
    </property>
    <property name="text">
     <string>アルコマーカーチェック
Check Aruco Marker</string>
    </property>
   </widget>
   <widget class="QPushButton" name="edgedetectbutton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>240</y>
      <width>231</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 238, 236);</string>
    </property>
    <property name="text">
     <string>エッジ認識設定 
Edge Detection Params</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quitbutton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>960</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 238, 236);</string>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>1901</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>1901</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>1901</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>1901</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>10</y>
      <width>321</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Uroob</family>
      <pointsize>41</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(238, 238, 236);</string>
    </property>
    <property name="text">
     <string>AI 検査</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>50</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Uroob</family>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(238, 238, 236);</string>
    </property>
    <property name="text">
     <string>by 設計課</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
